bundles/GraphicServiceOrder/Service/OrderService.php

```diff
@@ -84,6 +147,7 @@ public function createOrder(GsOrder $gsOrder, $uploadedFiles)
         // Notify messenger of new job.
         $this->messageBus->dispatch(new OwnCloudShareMessage($gsOrder->getId()));
 
+        $this->updateUser($gsOrder);
```

Comment:
Why are we updating the user here? Is it to help the user keep his information current?

Reply:
Yes. This way neither customer nor admin have to actively maintain user metadata between orders.